ddr/lib/ddr-scanner/dwarf/DwarfScanner.cpp

```diff
@@ -138,14 +143,25 @@ ddr_dw_finish(
 static int
 ddr_dw_init(
 	int           fd,
+	const char *filepath,
 	Dwarf_Handler errhand,
 	Dwarf_Ptr     errarg,
 	Dwarf_Debug  *dbg,
 	Dwarf_Error  *error)
 {
+#if defined(J9ZOS390) && !defined(OMR_EBCDIC) && defined(__open_xl__)
+	char *filename = a2e_string(filepath);
+	int rc = dwarf_goff_init_with_GOFF_filename(filename, errhand, errarg, 0, dbg, error);
+	if ( NULL != filename) {
+		free(filename);
+	}
```

Comment:
I don't expect it's safe to call `dwarf_goff_init_with_GOFF_filename()` with `NULL` as the first argument; formatting; name choice:
```c
	int result = DW_DLE_MAF;
	char *e_filepath = a2e_string(filepath);

	if (NULL != e_filepath) {
		result = dwarf_goff_init_with_GOFF_filename(e_filepath, errhand, errarg, 0, dbg, error)
		free(e_filepath);
	}
```

Reply:
Looks like this doesnt compile ...
`line 152.22: CCN5274 (S) The name lookup for "DW_DLE_MAF" did not find a declaration.`

Can we use this instead?
```
	int result = DW_DLV_ERROR;

#if defined(J9ZOS390) && defined(__open_xl__) && !defined(OMR_EBCDIC)
	char *e_filepath = a2e_string(filepath);

	if ( NULL != e_filepath) {
		result = dwarf_goff_init_with_GOFF_filename(e_filepath, errhand, errarg, 0, dbg, error);
		free(e_filepath);
	}
#else /* defined(J9ZOS390) && defined(__open_xl__) && !defined(OMR_EBCDIC) */
	Dwarf_Unsigned access = DW_DLC_READ;

	result = dwarf_init(fd, access, errhand, errarg, dbg, error);
#endif /* defined(J9ZOS390) && defined(__open_xl__) && !defined(OMR_EBCDIC) */

	return result;
```